### DISCUSSION TITLE
Cannot set shortcut

### DISCUSSION THREAD

**Discussion by camsloanftc** [rxhanson/Rectangle#824] [General]
I had "Next display" set as:

 **cmd + alt + shift + ctrl + backspace**

My computer restarted after an auto-update this morning, and now I cannot use this shortcut (and when I try to re-enter the shortcut it doesn't accept it). I am not sure if something took over this shortcut so now it is conflicting. Is there any easy way to find out a conflicting shortcut?

I was able to set it to cmd + alt + shift + [something else], so it seems to be struggling with the delete key, though I was also able to set it to use only cmd + delete with no problem.

I tried this on both a moonlander keyboard and the macbook pro keyboard with no luck.

I also checked "remove shortcut restrictions" but also no luck.

macOS version: 12.4
Rectangle version: .56
Logs if applicable (In Rectangle menu, hold option, "View Logging..."): none.

**Comment by rxhanson** [comment]
This does mean that the shortcut is somehow conflicting, and unfortunately macOS doesn't provide a means to figure out what has registered that shortcut as far as I know. 

I'm unable to register that keyboard shortcut, too. I'll see if I can figure out why.

**Reply by camsloanftc** [reply]
Thanks! I quickly dug around through the keyboard shortcuts menu on Mac OS to see if they had registered anything, but no luck so far. 

I will also report back if I find anything. At least now I know it's not just my machine.
